x/xvalidator/validator.go

```diff
@@ -8,23 +8,28 @@ import (
 	validator "gopkg.in/go-playground/validator.v9"
 )
 
+const (
+	minPort = 1
+	maxPort = 65535
+
+	portSeparator = ":"
+)
+
 // IsDomainName validates if given field is valid domain name.
 func IsDomainName(fl validator.FieldLevel) bool {
```

Comment:
@krhubert The following code was necessary. Please don't remove if you don't test.
```
	if fl.Field().String() == "" {	
		return true	
	}
```

Reply:
The following code is wrong and should not be here. It's not valid domain.

You can check the default validators here - https://github.com/go-playground/validator/blob/v9.27.0/baked_in.go#L64

None of them check for empty strings. If here check appears then something else is wrong.

Reply:
👍 condition removed and tag `omitempty` added to sid